runing/repositories.py

```
@@ -0,0 +1,54 @@
+from sqlalchemy.orm import Session
+from typing import List, Optional
+import models
+import schemas
+
+class UserRepository:
+    def __init__(self, db: Session):
+        self.db = db 
+        
+    def get_user(self, user_id: int) -> Optional[models.User]:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)